### PR TITLE
Booking page stage2

### PR DESCRIPTION
**Pull request by Falya**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,28 +10,28 @@
   "scripts": {
     "start": "node --experimental-modules ./bin/www",
     "server": "node --experimental-modules server.mjs",
-    "local": "nodemon --experimental-modules server.mjs "
+    "local": "nodemon --experimental-modules  server.mjs "
```
Comment (dzmitry-penda) [inline package.json]: Unneeded space

File: src/methods/clientMethods.mjs [hunk 2]
```diff
@@ -137,17 +138,58 @@ export async function getOptionsForFilters(params) {
...
   };
 }
+
+export async function toBlockSeat(params) {
+  const { seanceId, seat, row, userId } = params;
+
+  try {
+    const blockedSeat = await BlockedSeats.findOne({ seanceId, seat, row });
+    if (blockedSeat) {
+      return { success: false, message: 'seat is already blocked' };
```
Comment (dzmitry-penda) [inline src/methods/clientMethods.mjs]: You can create sort of dictionary of such error nmessgaes in separate file

File: src/models/BlockedSeats.mjs [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    ref: 'users',
+    required: true,
+  },
+  seatType: String,
+  expireAt: {
+    type: Date,
+    expires: 0,
+  },
+});
+// blockedSeatsSchema.index({ createdAt: 1 }, { expireAfterSeconds: 60 });
```
Comment (dzmitry-penda) [inline src/models/BlockedSeats.mjs]: should this be commented-out?

File: src/routes/api.mjs [hunk 4]
```diff
@@ -62,6 +59,34 @@ router.get('/seance/', (req, res) => {
...
+    .catch(err => console.log(err));
+});
+
+router.post('/seance/to-block-seat', passport.authenticate('jwt', { session: false }), async (req, res) => {
+  const params = {
+    ...req.body,
+    userId: req.user._id,
+  };
+  toBlockSeat(params).then(result => res.json(result));
+  // res.json(req.body);
```
Comment (dzmitry-penda) [inline src/routes/api.mjs]: Should this be commented out?

File: package.json (hunk 1 above)
Comment (Falya) [reply package.json]: fixed

File: src/models/BlockedSeats.mjs (hunk 3 above)
Comment (Falya) [reply src/models/BlockedSeats.mjs]: fixed

File: src/routes/api.mjs (hunk 4 above)
Comment (Falya) [reply src/routes/api.mjs]: No, I just forgot to remove it.

File: src/methods/clientMethods.mjs (hunk 2 above)
Comment (Falya) [reply src/methods/clientMethods.mjs]: fixed

File: package.json [hunk 5]
```diff
@@ -17,21 +17,21 @@
...
     "express": "~4.16.1",
     "http-errors": "~1.6.3",
     "jsonwebtoken": "^8.5.1",
     "mongoose": "^5.5.11",
+    "morgan": "^1.9.1",
     "passport": "^0.4.0",
     "passport-jwt": "^4.0.0",
-    "pug": "2.0.0-beta11",
-    "morgan": "^1.9.1"
+    "pug": "2.0.0-beta11"
```
Comment (dzmitry-penda) [inline package.json]: why do you need pug?

File: src/namedMessages/namedMessages.mjs [hunk 6]
```diff
@@ -0,0 +1,20 @@
+const messages = {
+  BLOCK_SEAT_SUCCESS: { success: false, message: 'seat is already blocked' },
```
Comment (dzmitry-penda) [inline src/namedMessages/namedMessages.mjs]: you can out here just strings, but the idea is right

File: package.json (hunk 5 above)
Comment (Falya) [reply package.json]: Sorry. I don`t need it. I just took it from example.

File: src/namedMessages/namedMessages.mjs (hunk 6 above)
Comment (Falya) [reply src/namedMessages/namedMessages.mjs]: fixed

File: src/methods/clientMethods.mjs [hunk 7]
```diff
@@ -137,17 +139,55 @@ export async function getOptionsForFilters(params) {
...
   const seance = await Seance.findById(seanceId);
   const movieTheater = await MovieTheater.findOne(
     { 'halls._id': seance.hallId },
     { halls: { $elemMatch: { _id: seance.hallId } } }
   ).select('-seances');
-  console.log('seance', seance);
-  console.log('movieTheater', movieTheater);
-
+  const nowTime = new Date();
+  nowTime.toISOString();
```
Comment (dzmitry-penda) [inline src/methods/clientMethods.mjs]: const nowTime = new Date();
  nowTime.toISOString();

Comment (dzmitry-penda) [reply src/methods/clientMethods.mjs]: absolutely useless

Comment (Falya) [reply src/methods/clientMethods.mjs]: Agree with you! Removed it

**Pull request merged**